app/main.py

```diff
@@ -1 +1,12 @@
 # write your code here
+def copy_file(command: str) -> None:
+
+    cp, source_file_name, copy_file_name = command.split()
+    if cp != "cp" or source_file_name == copy_file_name:
+        return
+
+    with (open(source_file_name, "r") as source,
+          open("copy_file_name.txt", "w") as backup):
+        backup.write(source.read())
+        source.close()
+        backup.close()
```

Comment:
The `close()` method calls for `source` and `backup` are unnecessary here because the `with` statement automatically handles the closing of files. You can safely remove these lines.